src/content/docs/en/developers/l1-and-l2-bridging.mdx

```diff
@@ -22,14 +23,29 @@ In addition to token transfers, the Scroll Messenger contract enables cross-chai
 
 ## L1 Gateway architecture
 
+
 <ClickToZoom src={L1GatewayWHITE} />
 
 There are many entry points from the user to the Scroll bridge. This will depend on what you want to do and how you want to do it. If you want to send ETH or ERC20 tokens, you should use the `GatewayRouter` . If you want to send NFTs, you should use the `L1ERC721Gateway` or `L1ERC1155Gateway`. If you want to send arbitrary data, you should use the `L1ScrollMessenger`. All Gateway transfers use the Scroll Messenger to send assets cross-chain, whose job is to append the transactions to the Message Queue for L2 inclusion.
 
+<Aside type="tip" title="Upgrade Notice">
+With the February 2024 Bridge Upgrade, minor changes will be made to the above flow. Most notably, `L1GatewayRouter` will now bypass `L1ETHGateway` when bridging ETH to improve gas efficiency.
+
+The upgrade is expected to be finalized on February 19th, 2024 after a two-week timelock. Scroll Sepolia has already undergone this upgrade. Read more [here](https://scroll.io/blog/to-do-slug).
```

Comment:
We'll probably need to change to Feb 21

Reply:
removed this, fixed all other date issues